docs/README.md

```diff
@@ -1,125 +1,76 @@
-
-> :loudspeaker: **Important:** After installation, all navigation and search pages will be delivered by the new VTEX Search.
->
-> Confirm that the [Indexing Process step](#indexing-process) has been completed successfully before promoting the app to `master`.
-
-### Indexing Process
+You are now able to use all of the blocks exported by the `search` app. Check the full list below:
 
-After installing our suite of apps, your workspace is ready to begin it's Search journey, but first we need to index your
-account's catalog in our Search Engine.
+| Block name | Description |
+| -------------- | ----------------------------------------------- |
+| `autocomplete-result-list.v2` | Provides customized autocomplete features in the search bar component, such as top searches, search history, product suggestions or term suggestions. You can read more about the Intelligent Search [autocomplete feature](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/4gXFsEWjF7QF7UtI2GAvhL) on VTEX Help Center. |
+| `search-banner` | Renders a customized banner according to the search query performed. |
+| `did-you-mean` | Helps users with possible misspelling corrections for the current search query. |
+| `search-suggestions` | Renders a list of similar search terms for the current search query. |
 
-Go to your Admin page and click on the menu [Search > Integration Settings](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb/6wKQgKmu2FT6084BJT7z5V)
-and start the Indexing Process.
+:warning: The blocks `banner`, `did-you-mean` and `suggestions` require you to have [Search Results app](https://vtex.io/docs/components/all/vtex.search-result@3.59.0/) version 3.x or higher installed in your theme.
 
-![](https://images.ctfassets.net/alneenqid6w5/6Dhh6x6Roi1vRePJDtbOFY/d642a9f13d421ef3f3062a5ff261ff39/config-int-EN.png)
+### Step 2 - Adding the Search's blocks into the theme
 
-### UI Components
-
-If you want to use our UI components you need import this app in your dependencies on the `manifest.json`.
+First, declare the `autocomplete-result-list.v2` block as a child block of the [`search-bar` block](https://vtex.io/docs/components/all/vtex.store-components/searchbar), exported by the Store Components app. For example:
 
 ```json
 {
-  "dependencies": {
-    "vtex.search": "1.x"
+  "search-bar": {
+    "blocks": ["autocomplete-result-list.v2"],
+    "props": {
+      "openAutocompleteOnFocus": true
+    }
   }
 }
 ```
 
-This app provides the following blocks:
-
-- [Autocomplete](#autocomplete)
-- [Banner](#search-result-complements)
-- [DidYouMean](#search-result-complements)
-- [Suggestions](#search-result-complements)
-
-#### Autocomplete
-
-We provide a customized autocomplete with new features. It includes:
+#### `autocomplete-result-list.v2` props
 
-- Top Searches.
-- Search History.
-- Product Suggestions.
-- Term Suggestions.
+>>>ADD PROP TABLE
```

Comment:
the idea here is to add the table containing the props for the autocomplete block. If there are no props, then we can just delete this h4.